docs/design/relationship-shared-library.adoc

```diff
@@ -29,24 +29,24 @@ Following are some details on what that means practically:
+At the moment, the orchestration pipeline only sends requests to the doc gen service, which is running in each namespace, and receives rendered documents. We believe that this architecture is needlessly complex for the Tekton approach. Instead of having a long-running service in each namespace which consumes resources and is hard to maintain (as every initiative needs to do that themselves), we could simply call the service classes from the doc gen services directly from the Groovy code of the new "release manager / doc gen task".
 
-## Quickstarters
+== Quickstarters
 
 Quickstarters currently provide a `Jenkinsfile.template` which will get rendered into the resulting component repository. With `ods-pipeline`, this would be replaced by an `ods.yml` file.
 
-Furhter, the quickstarter provisioning itself is also done by a Jenkins pipline, based on the `Jenkinsfile` in the quickstarter. This also needs to be replaced with something. Right now, the quickstart process consists of creating a repo, and creating OpenShift resources. I would stop creating OpenShift resources in the new world, and simply require each quickstarter to supply a Helm chart that is then deployed automatically with the first pipeline run (we need to be infrastructure-as-code with Helm as no magic export functionality exists like with Tailor). With that in mind, we only need to create a repo. Often, that step includes running tech-specific tasks (e.g. generators). I suggest that quickstarter authors define a TaskRun, which then gets executed. That way they can pick their own container image and do whatever they want, in whatever language they want.
+Further, the quickstarter provisioning itself is also done by a Jenkins pipeline, based on the `Jenkinsfile` in the quickstarter. This also needs to be replaced with something. Right now, the quickstart process consists of creating a repo, and creating OpenShift resources. We would stop creating OpenShift resources in the new world, and simply require each quickstarter to supply a Helm chart that is then deployed automatically with the first pipeline run (we need to be infrastructure-as-code with Helm as no magic export functionality exists like with Tailor). With that in mind, we only need to create a repo. Often, that step includes running tech-specific tasks (e.g. generators). We suggest that quickstarter authors define a TaskRun, which then gets executed. That way they can pick their own container image and do whatever they want, in whatever language they want.
```

Comment:
isn't it "We suggest that quickstarter authors define a Task" instead of "We suggest that quickstarter authors define a TaskRun"? maybe did not understand

Reply:
I think in this context a `TaskRun` might be what we want.

That said, arguably the whole section should be removed as the relationship to quickstarters is pretty unclear at the moment. That's leave it like it is for now, and I'll potentially move it to the wiki at some point.

Reply:
@michaelsauter would prefer to leave some context in, as I plan to bring my team in to collaborate, e.g. on this topic with you